Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -121,10 +121,10 @@ public PrintHelper(Panel canvasContainer, PrintHelperOptions defaultPrintHelperO
         }
 
         /// <summary>
-        /// Add an element to the list of printable elements.
+        /// Adds an element to the list of elements to print.
```

Comment:
`printable elements` -> `elements to print`

is this correct?

Reply:
Yes, and even more accurate.